parsers/AR.py

```diff
@@ -143,8 +153,46 @@ def fetch_exchange(
     logger: Logger = getLogger(__name__),
 ) -> dict:
     """Requests the last known power exchange (in MW) between two zones."""
+    if target_datetime:
+        raise NotImplementedError("This parser is not yet able to parse past dates")
+
+    sorted_zone_keys = sorted([zone_key1, zone_key2])
+    sorted_codes = "->".join(sorted_zone_keys)
+    flow = 0
+    target_datetime = ""
```

Comment:
Shouldn't the default for this be an actual time? Pretty sure `arrow.get(target_datetime).datetime` would fail if target_datetime is `null`. Let's change this to be the current time